doc/source/release-notes/Percona-Server-5.7.34-37.rst

```diff
@@ -0,0 +1,26 @@
+.. _PS-5.7.34-37:
+
+================================================================================
+*Percona Server for MySQL* 5.7.34-37
+================================================================================
+:Date: May 26, 2020
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/5.7/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.7.34-37
+includes all the features and bug fixes available in
+`MySQL 5.7.34 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-34.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-4497`: Incorrect option error message for mysqlbinlog
+* :jirabug:`PS-7498`: Prevent the replication coordinator thread stuck in Waiting until MASTER_DELAY seconds after master executed event while handling partial relay log transactions.
+* :jirabug:`PS-7566`: Correct version matching in RPM spec changelog for PS packages
+* :jirabug:`PS-7574`: Missing xml file with MTR test results on Jenkins
+* :jirabug:`PS-7593`: When changing the tx-isolation on a session, after a transaction has executed, the change is not honored and may cause a service failure.
+* :jirabug:`PS-7636`: Fix CPU features detection in MyRocks
+* :jirabug:`PS-7651`: Travis CI and Azure: add clang-12 testing
```

Comment:
PS-7651 and PS-7690 are Admin & Maint, internal only

Reply:
PS-7651 removed.